### ISSUE TITLE
Django 1.8 breaks rest-auth

### ISSUE THREAD

**Issue by philippeluickx** [Tivix/django-rest-auth#73]
See https://github.com/pennersr/django-allauth/issues/858#issuecomment-92357122

There is an invocation of the user when the account is not setup yet.

```python
        try:
            login = self.adapter.complete_login(request, app, token,
                                                response=access_token)
            token.account = login.account
            login.token = token
            complete_social_login(request, login)
        except HTTPError:
            raise serializers.ValidationError('Incorrect value')
```

**Comment by mjaworski** [comment]
Fixed in #75

**Comment by philippeluickx** [comment]
Awesome, thanks!
